packages/@okta/vuepress-site/docs/guides/common-hook-set-up-steps/adding-basic-auth/index.md

```diff
@@ -15,7 +15,9 @@ To add the `npm` packages:
 
 After including the packages, add the following code snippet in your project.
 
-> **Note:** As part of the hook configuration process, you add the authorization field and secret in your Okta org, which inputs this content into the Okta request header. The authorization and secret are authorized by your external service.
+As part of the Inline Hook configuration process, you add the authorization field and secret in your Okta org, which inputs this content into the Okta request header. The authorization and secret are authorized by your external service.
+
+The following Inline Hook examples use the Basic HTTP authentication scheme, which encodes the user and password pair in Base64. Make sure to also add the scheme `Basic` to the Base64 secret value, for example: `Basic YWRtaW46c3VwZXJzZWNyZXQ=`. See [HTTP Basic Authentication](/books/api-security/authn/api-authentication-options/#http-basic-authentication) for further information.
```

Comment:
> .. which encodes the username and password ... [below you mention that the string is username:password, so we should be consistent in the description here.]

> Ensure that you include the `Basic` authentication scheme as part of the Base64 secret `value` parameter. For example, `Basic YWRtaW46c3VwZXJzZWNyZXQ=`. See [HTTP Basic Authentication](/books/api-security/authn/api-authentication-options/#http-basic-authentication)~ for further information~.

(changed to style guidelines for "ensure" vs. "make sure" and for hyperlink text).

This is a very minor quibble, but is the '=' at the end of that value meant to signal an equals sign, or is it just a special character that is part of the example value? If it's just the latter, maybe tweak it to be another alphanumeric character to avoid confusion (like from n00bs like me).

Reply:
Thanks, updated as per your other comment. the `=` is part of the Base64 encoded value, which I explained in my re-write.